prevents redundant memcached compression (#39747)
rails/rails: activesupport/CHANGELOG.md

> *   Fixes issue in `ActiveSupport::Cache::MemCacheStore` which can cause duplicate compression,
>     or cause the provided `compression_threshold` to not be respected.

Reviewer: The changelog entry is for a bug that's now fixed, let's describe it in the past tense.

```suggestion
*   Fix an issue in `ActiveSupport::Cache::MemCacheStore` which caused duplicate compression,
    and caused the provided `compression_threshold` to not always be respected.
```
